config/routes.rb

```diff
@@ -1,9 +1,9 @@
 Rails.application.routes.draw do
   
-  resources :cards
-
   root 'home#index'
-
+  post '/' => 'home#check_card'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.